app/templates/webpack.config.js

```diff
@@ -1,14 +1,22 @@
 const path = require('path');
 const webpack = require('webpack');
-const UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
 
 const libraryName = '<%= libraryNameCamelized %>';
+const isProduction = process.env.NODE_ENV === 'production';
+
+const UglifyJsPlugin = webpack.optimize.UglifyJsPlugin;
+const extractStyles = new ExtractTextPlugin({
```

Comment:
Looks like this file is missing an import for `ExtractTextPlugin`.

Reply:
Fixed